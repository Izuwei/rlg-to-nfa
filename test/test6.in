A,B,C
a,b,c
A
A->aA
A->bB
A->#
B->bB
B->cC
B->#
C->cC
C->#
