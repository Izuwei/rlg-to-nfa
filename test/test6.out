1,2,3
abc
1
1,2,3
1,a,1
1,b,2
2,b,2
2,c,3
3,c,3
